pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -3088,8 +3088,13 @@ public void onUpdate(TopicPolicies policies) {
         }
 
         topicPolicies.getMaxSubscriptionsPerTopic().updateTopicValue(policies.getMaxSubscriptionsPerTopic());
-
         topicPolicies.getInactiveTopicPolicies().updateTopicValue(policies.getInactiveTopicPolicies());
+        Arrays.stream(BacklogQuotaType.values()).forEach(type ->
+                this.topicPolicies.getBackLogQuotaMap().get(type).updateNamespaceValue(
```

Comment:
Why do we update the value of Namespace to null in TopicPolicies update?

Reply:
Set to null means user have deleted the namespace level setting.

Reply:
However, this method is only triggered when the Topic-level Policies are updated. Why do we update the Namespace-level Policies in this method?

Reply:
OMG. Nice catch.  Fixed.